### DISCUSSION TITLE
I'm not able to write the code Logger.error()

### DISCUSSION THREAD

**Discussion by rohrat18** [jongpie/NebulaLogger#725] [Q&A] [answered]
Its giving me the error saying - Method does not exist or incorrect signature: void error(String) from the type Logger
please let me know what im missing

![Github error](https://github.com/user-attachments/assets/4529b092-e3d1-4733-b1ff-461783e5ce0a)

**Comment by jongpie** [comment]
@rohrat18 based on that error, it looks like you have a class called `Logger`, but it's not from Nebula Logger. Try looking under Setup --> Installed Packages to see if you have Nebula Logger installed.

**Reply by rohrat18** [reply]
![nebula Logger installed](https://github.com/user-attachments/assets/bd9ac18d-3652-43b6-b80b-b04db0c23eb7)
I have installed the managed version and also I'm not able to install the unlocked package getting these errors
![unlock package failed](https://github.com/user-attachments/assets/d2e189ee-aa2a-4245-ab0c-39c33e9f6a0b)

**Reply by jongpie** [reply, marked as answer]
Since you already have an existing `Logger` class in your org, you won't be able to use the unlocked package. 

To use the managed package, you'll need to include the `Nebula` namespace, like:

```apex
Nebula.Logger.error('example error log entry');
Nebula.Logger.saveLog();
```

**Reply by rohrat18** [reply]
Thanks for the help @jongpie
